src/main/java/org/myrobotlab/service/LocalSpeech.java

```diff
@@ -107,6 +107,10 @@ public AudioData generateAudioData(AudioData audioData, String toSpeak) throws I
     
 
     Platform platform = Runtime.getPlatform();
+    if (ttsCommand == null) {
+      log.error("tts command is invalid");
+      return null;

```

Comment:
Are there null checks on whatever calls this method? Returning null could be a problem if not

Reply:
the AbstractSpeechSynthesis doesn't consume it, it only relays it.
If you don't get a AudioData back, audio data wasn't generated, imo, a fair contract